README.md

```diff
@@ -1,4 +1,5 @@
 infamous
+[![All Contributors](https://img.shields.io/badge/all_contributors-1-orange.svg?style=flat-square)](#contributors)
```

Comment:
Maybe just "contributors". WDYT?

Reply:
I personally don't have an issue with dropping the "all", though that part was provided by the cli tool. I'm not sure if that's just from an init step or if it replaces it on each update. If the latter, we'd have to work around that in some way (e.g. contribute to their project, manually change it, write a script to correct it after running the cli tool).

Reply:
If the latter, we can just leave it like that then, no biggie. I'm guessing it was probably on init. Yeah, forking it and using our forked version of the CLI tool would work too, but maybe too much work.